central_instances.py

```diff
@@ -0,0 +1,1250 @@
+#!/usr/bin/env python
```

Comment:
This script should go to the `try` directory, and possibly be consolidated with the other script.